common/djangoapps/django_comment_common/tests.py

```diff
@@ -1,9 +1,9 @@
+from django.test import TestCase
 from nose.plugins.attrib import attr
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
 
-from django.test import TestCase
```

Comment:
Did isort change the order of these imports? If not, please revert the change to this file.